build.rs

```diff
@@ -1,7 +1,21 @@
-use vergen::{Config, ShaKind};
+use std::process::Command;
+
+const GIT_HASH_ENV_VAR: &str = "GIT_SHA_SHORT";
 
 fn main() {
-    let mut config = Config::default();
-    *config.git_mut().sha_kind_mut() = ShaKind::Short;
-    let _ = vergen::vergen(config);
+    //Build command to retrieve the short git commit hash
```

Comment:
Can you please pull those changes ? to avoid a git dependency 

https://github.com/casper-ecosystem/casper-client-rs/pull/175/files

as per https://github.com/casper-ecosystem/casper-client-rs/issues/163